staging/src/k8s.io/apiserver/pkg/server/config.go

```diff
@@ -588,7 +588,7 @@ func eventReference() (*corev1.ObjectReference, error) {
 	}
 	if len(ns) == 0 {
 		pod = ""
-		ns = "kube-system"
+		ns = "openshift-kube-apiserver"
```

Comment:
@s-urbaniak @sttts using kube-system through off an event integration test.  Also, we know that our kube-apiserver is never in kube-system.  Oddly, this entire method appears to be carried (stefan I think?), but it's not in a patch file.

Reply:
:+1: for changing this.

@mfojtik is this your code?

Reply:
@sttts negative